src/main/java/seedu/medibook/model/Account.java

```diff
@@ -0,0 +1,56 @@
+package seedu.medibook.model;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.medibook.model.doctor.Doctor;
+
+public class Account {
+
+    private String username;
+    private String password;
+    private Doctor doctor;
+
+    /**
+     * Constructs an account with a username and a password.
+     */
+    public Account(String username, String password, Doctor doctor) {
+        requireNonNull(doctor);
+        this.username = username;
+        this.password = password;
+        this.doctor = doctor;
+    }
```

Comment:
Maybe it is better to handle the exception of empty string "" as username/password?